packages/browserslist-config/CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines and [LMC GitHub preset](https://github.com/lmc-eu/code-quality-tools/tree/main/packages/conventional-changelog-lmc-github) for a list of supported types.
+
+<a name="1.0.1"></a>
+
+## [1.0.1](https://github.com/lmc-eu/code-quality-tools/compare/@lmc-eu/browserslist-config@1.0.0...@lmc-eu/browserslist-config@1.0.1) (2021-12-18)
```

Comment:
Cool, link to diff is useful! Maybe I'd also expect a link to GitHub Releases page here. But I'm not sure it's applicable for monorepos.